packages/pfsense/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.23.1"
```

```suggestion
- version: "1.24.0"
```
also, need to update `version` in `manifest.yml`